tests/suites/test_suite_x509parse.function

```diff
@@ -1250,6 +1250,86 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE depends_on:MBEDTLS_FS_IO:MBEDTLS_X509_CRT_PARSE_C:MBEDTLS_CERTS_C:MBEDTLS_SHA1_C:MBEDTLS_RSA_C */
+void mbedtls_x509_crt_parse_subjectkeyid( char * crt_path, char *subjectKeyId, int subjectKeyIdLength, int retVal )
+{
+    mbedtls_x509_crt crt;
+    int i = 0;
+    int result = 0;
+
+    mbedtls_x509_crt_init( &crt );
+
+    TEST_ASSERT( (mbedtls_x509_crt_parse_file( &crt, crt_path )& 0xFFFF) == retVal );
+
+    if(retVal != 0)
```

Comment:
needs spaces inside round brackets, i.e. "if( retVal != 0 )"

Reply:
FYI: please see https://tls.mbed.org/kb/development/mbedtls-coding-standards for detailed guidance

Reply:
Thank you for the link! I have checked it and will use it in the future. 
All the missing whitespaces have been added

Reply:
Corrected